Sources/ClientRuntime/Networking/Http/HttpBody.swift

```diff
@@ -16,11 +16,7 @@ extension HttpBody: Equatable {
         case (.data(let lhsData), .data(let rhsData)):
             return lhsData == rhsData
         case (.stream(let lhsStream), .stream(let rhsStream)):
-            do {
-                return try lhsStream.isEqual(to: rhsStream)
-            } catch {
-                return false
-            }
+            return lhsStream === rhsStream
```

Comment:
Stream equatability isn't used for any purpose other than to allow streams to be part of input & output shapes, which are `Equatable` by default.  The old equality implementation for streams synchronously read them to the end, which was unnecessary & added complexity.  So I have changed the stream equality check to a simple pointer comparison.

Reply:
As it turns out, this did require a change to protocol tests to compare stream contents in testing (see below). But I'd rather have this code in tests than in production.